users/src/controllers/user.controller.ts

```diff
@@ -152,4 +164,65 @@ export const findRelation = async (req: Request, res: Response) => {
         status: relation
     }
     res.status(200).send(response);
+}
+
+/**
+ * Method for editing any field of the user with the specified id.
+ * Expects req.body to have a user object
+ * If all checks pass, will update User with the fields specified
+ * Then sends 200 alongside the newly updated user object.
+ * @param req 
+ * @param res 
+ */
+export const editUser = async (req: Request, res: Response) => {
+    if(!req.currentUser!.id) throw new BadRequestError("Please enter a user ID!");
+    console.log(req.currentUser!.id);
+    const user = req.user;
+    if(!user) throw new NotFoundError();
+    // if(isEmpty(req.body)) throw new BadRequestError("Empty request!");
+    if (req.body.year) {
```

Comment:
this logic needs to be removed, see createUser for more detail

Reply:
which one req.body.year or the !user

Reply:
both